lib/go-tc/deliveryservices.go

```diff
@@ -49,6 +49,14 @@ const MaxRangeSliceBlockSize = 33554432
 // values of a Delivery Service's 'Active' property (v3.0+).
 type DeliveryServiceActiveState string
 
+// These names of URL query string parameters are not allowed to be in a
+// Delivery Service's "ConsistentHashQueryParams" set, because they collide with
+// query string parameters reserved for use by Traffic Router.
+const (
+	ReservedConsistentHashingQueryParameterFormat = "format"
+	ReservedConsistentHashingQueryParameterTRRED  = "trred"
```

Comment:
Should probably include `"fakeClientIpAddress"`, too

https://github.com/apache/trafficcontrol/blob/7741530c494d797758d226885abee3ed7feda7be/traffic_router/core/src/main/java/org/apache/traffic_control/traffic_router/core/request/HTTPRequest.java#L30